source/wp-content/themes/wporg-documentation-2022/functions.php

```diff
@@ -46,11 +46,69 @@ function set_site_breadcrumbs( $breadcrumbs ) {
 	if ( is_front_page() ) {
 		return array();
 	}
-
 	$breadcrumbs[0]['title'] = __( 'Documentation', 'wporg-docs' );
+
+	if ( is_category() ) {
+		// Format: home / topic page / category
+		$category    = get_queried_object();
+		$breadcrumbs = array( $breadcrumbs[0] );
+		if ( $category->category_parent ) {
+			$parent        = get_term( $category->category_parent );
+			$breadcrumbs[] = array(
+				'url'   => get_topic_permalink( $parent ),
+				'title' => $parent->name,
+			);
+		}
+		$breadcrumbs[] = array(
+			'url'   => '',
+			'title' => $category->name,
+		);
+	} elseif ( is_singular( 'helphub_article' ) ) {
+		// Format: home / topic page / category / article title
+		$breadcrumbs = array( $breadcrumbs[0] );
+		$categories  = get_the_category();
+		if ( $categories ) {
+			$category = $categories[0];
+			if ( $category->parent ) {
+				$parent        = get_term( $category->parent );
+				$breadcrumbs[] = array(
+					'url'   => get_topic_permalink( $parent ),
+					'title' => $parent->name,
+				);
+			}
+			$breadcrumbs[] = array(
+				'url'   => get_term_link( $category->term_id, $category->taxonomy ),
+				'title' => $category->name,
+			);
+		}
+		$breadcrumbs[] = array(
+			'url'   => '',
+			'title' => get_the_title(),
+		);
+	}
+
 	return $breadcrumbs;
 }
 
+/**
+ * Get the topic landing page permalink for a given parent category.
+ */
+function get_topic_permalink( $category ) {
+	if ( empty( $category->slug ) ) {
+		return '';
+	}
+	switch ( $category->slug ) {
+		case 'wordpress-overview':
+			return site_url( '/overview/' );
+		case 'technical-guides':
+			return site_url( '/technical-guides/' );
+		case 'support-guides':
+			return site_url( '/support-guides/' );
+		case 'customization':
+			return site_url( '/customization/' );
+	}
```

Comment:
Not sure if we're expecting more outliers here, but could be simplified

```suggestion
        if ( 'wordpress-overview' === $category->slug ) {
	    return site_url( '/overview/' );
	}
	
	return site_url( '/' . $category->slug . '/' );
```

Reply:
These pages haven't been created yet, and while I'm pretty sure this is what the URLs & slugs will be, I'm not positive. I think your suggestion would be okay, though.